src/integrations/contact-loaders/index.js

```diff
@@ -75,10 +75,15 @@ export async function getIngestMethod(name, organization, user) {
 }
 
 export async function getAvailableIngestMethods(organization, user) {
+  const enabledIngestMethods = (
+    getConfig("CONTACT_LOADERS", organization) ||
+    "csv-upload,test-fakedata,datawarehouse"
+  ).split(",");
+
   const ingestMethods = await Promise.all(
-    Object.keys(CONFIGURED_INGEST_METHODS).map(name =>
-      getIngestMethod(name, organization, user)
-    )
+    enabledIngestMethods
```

Comment:
@schuyler1d please look at line 26 in this file. Would CONFIGURED_INGEST_METHODS need to be a superset across all organizations?

Reply:
correct.